web/modules/custom/hoeringsportal_audit_log/src/Form/SettingsForm.php

```diff
@@ -80,11 +82,12 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#tree' => TRUE,
     ];
 
-    $form['logged_pages']['logged_route_names'] = [
+    $routesToLog = $config->get('routes_to_log') ? Yaml::dump($config->get('routes_to_log')) : NULL;
+    $form['logged_pages']['routes_to_log'] = [
       '#type' => 'textarea',
       '#title' => $this->t('Route names'),
-      '#default_value' => $config->get('logged_route_names') ? $this->fromArrayToString($config->get('logged_route_names')) : NULL,
-      '#description' => $this->t('Route names (one per line) to log when users visit, they can look like this: <code>hoeringsportal_citizen_proposal.admin_supporter</code> or <code>node.add</code>, if in doubt, ask your friendly neighborhood programmer.'),
+      '#default_value' => $routesToLog,
+      '#description' => $this->t("Write the configuration in YAML. We log when a user visits something matching the routes or the url patterns. The url patterns can look like this: <code>'/^\/admin\/content\?title=&type=All&status=1$/'</code>. The routes can look like this: <code>hoeringsportal_citizen_proposal.admin_supporter</code>. If you find yourself in doubt on how to fill this text area, ask your friendly neighborhood programmer. With great power comes great responsibility."),
```

Comment:
Consider adding a full example (not tested):

```suggestion
      '#description' => $this->t("Write the configuration in YAML. We log when a user visits something matching the routes or the url patterns. The url patterns can look like this: <code>@example</code>. The routes can look like this: <code>hoeringsportal_citizen_proposal.admin_supporter</code>. If you find yourself in doubt on how to fill this text area, ask your friendly neighborhood programmer. With great power comes great responsibility.", [
      '@example' => <<'YAML'
- '…'
- '@/admin/hest'      
YAML,
      ]),
```